DependencyInjection/Configuration.php

```diff
@@ -724,6 +724,7 @@ private function getOrmCacheDriverNode(string $name): ArrayNodeDefinition
                 ->scalarNode('type')->defaultNull()->end()
                 ->scalarNode('id')->end()
                 ->scalarNode('pool')->end()
+                ->scalarNode('psr6')->defaultFalse()->end()
```

Comment:
Is there a way around adding the option?

Reply:
Apologies, this slipped in from during the refactoring. I've removed it.